homeassistant/components/overkiz/climate/somfy_heating_temperature_interface.py

```diff
@@ -110,9 +110,14 @@ def hvac_mode(self) -> HVACMode:
 
     async def async_set_hvac_mode(self, hvac_mode: HVACMode) -> None:
         """Set new target hvac mode."""
-        await self.executor.async_execute_command(
-            OverkizCommand.SET_ACTIVE_MODE, HVAC_MODES_TO_OVERKIZ[hvac_mode]
-        )
+        if hvac_mode == HVACMode.OFF:
```

Comment:
```suggestion
        if hvac_mode is HVACMode.OFF:
```
compare enums with `is`